### ISSUE TITLE
Connections and Config pane typing issues in Windows 10

### ISSUE THREAD

**Issue by alexbransky** [mRemoteNG/mRemoteNG#1254]
There is an issue with mRemoteNG where the user becomes unable to type in the Connections and Config panes in Windows 10.

## Expected Behavior
We should always be able to type in the Connections and Config panes.

## Current Behavior
I create a new connection in the Connections pane but can't rename it because typing does nothing.  Then I can try to change the name and hostname in the Config pane and can't type there either.  I'm not sure how to repro but it's happened a few times over the past month and restarting mRemoteNG fixes the problem.  

## Possible Solution
Unknown.

## Steps to Reproduce (for bugs)
Unknown.

## Context
It has been annoying but not so much that I won't still use the application.

## Your Environment
* Version used: 1.76.11.40527
* Operating System and version (e.g. Windows 10 1709 x64): Windows 10 Enterprise 1607 x64, OS Build 14393.2248 with 64 GB of RAM

**Comment by mjmooney** [comment]
Is it a focus issue where the application loses focus (check the title bar for color solid or not on W10).  I know I get focus issues where I have to select somewhere on titlte / panel bars before I'm able to ineract with the conn/cfg panels again.  I'm trying to find an issue that referenced this, I know I've seen it prior.

**Comment by alexbransky** [comment]
Nope, I can see the cursor is in the box where it should be, but typing does nothing.

**Comment by Kvarkas** [comment]
not relevant
